cmd/snap/cmd_interfaces.go

```diff
@@ -37,19 +37,19 @@ var shortInterfacesHelp = i18n.G("Lists interfaces in the system")
 var longInterfacesHelp = i18n.G(`
 The interfaces command lists interfaces available in the system.
 
-By default all plugs and slots, used and offered by all snaps, are displayed.
+By default all slots and plugs, used and offered by all snaps, are displayed.
  
-$ snap interfaces <snap>:<plug or slot>
+$ snap interfaces <snap>:<slot or plug>
 
-Lists only the specified plug or slot.
+Lists only the specified slot or plug.
 
 $ snap interfaces <snap>
 
-Lists the plugs offered and slots used by the specified snap.
+Lists the slots offered and plugs used by the specified snap.
 
 $ snap interfaces --i=<interface> [<snap>]
 
-Lists only plugs and slots of the specific interface.
+Lists only slots and plugs of the specific interface.
```

Comment:
Can you please replace this whole text by what's in the spec, just swapping plugs and slots?

It's much easier to understand than what's here.